solana/pyth2wormhole/client/src/lib.rs

```diff
@@ -402,8 +399,8 @@ pub fn gen_attest_tx(
 pub async fn crawl_pyth_mapping(
     rpc_client: &RpcClient,
     first_mapping_addr: &Pubkey,
-) -> Result<HashMap<Pubkey, HashSet<Pubkey>>, ErrBox> {
-    let mut ret = HashMap::new();
+) -> Result<Vec<P2WProductAccount>, ErrBox> {
```

Comment:
adding the name of each product to the result here (so we can match on it to make the batches)